core/src/main/java/io/grpc/internal/DelayedClientTransport.java

```diff
@@ -277,6 +368,13 @@ private void createRealStream(ClientTransport transport) {
       setStream(transport.newStream(method, headers, callOptions));
     }
 
+    /**
+     * Indicates whether the pending stream is with fail fast call option.
+     */
+    public boolean isFailFast() {
+      return callOptions.isFailFast();
+    }
```

Comment:
Nit: callOptions can be accessed from the outer class, so you just call `stream.callOptions.isFailfast()` thus don't need this method.